include/uapi/sound/sof/ext_manifest.h

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ */
+
+/*
+ * Extended manifest is a place to store metadata about firmware, known during
+ * compilation time - for example firmware version or used compiler.
+ * Given information are read on host side before firmware startup.
+ * This part of output binary is not signed.
+ */
+
+#ifndef __KERNEL_EXT_MANIFEST_H__
+#define __KERNEL_EXT_MANIFEST_H__
```

Comment:
SOF_FIRMWARE_EXT_MANIFEST

This is not a kernel manifest really

Reply:
changed